pkg/features/features.go

```diff
@@ -91,6 +91,12 @@ const (
 	// alpha: v1.121.0
 	// beta: v1.123.0
 	DoNotCopyBackupCredentials featuregate.Feature = "DoNotCopyBackupCredentials"
+
+	// OpenTelemetryCollector enables the usage of an OpenTelemetry Collector instance in the Control Plane of Shoot clusters.
+	// All logs will be routed through the Collector before they reach the Vali instance.
+	// owner: @rrhubenov
+	// alpha: v1.123.0
```

Comment:
```suggestion
	// alpha: v1.124.0
```

as https://github.com/gardener/gardener/pull/12475 was meanwhile merged.